### ISSUE TITLE
mouseout event not triggering properly

### ISSUE THREAD

**Issue by justenPalmer** [zeroclipboard/zeroclipboard#559]
In version 2.2.0, I am using zero clipboard with the following config:

{forceHandCursor:true,bubbleEvents:true}

I found that the mouseout event was not being propagated properly.  Basically what would happen is the flash element would correctly pass through the mouseenter event - my button would receive the zero clipboard hover class.  However, when the user left the element the hover class was not being removed - leaving my element in the hover state.  I fixed this issue by changing the event '_mouseout' in the switch statement around line 1000 to 'mouseout'.  It now works properly, but I'm not sure about the implications of the change.

**Comment by JamesMGreene** [comment]
Yeah, there is a lot going on here that I was trying to sort out but my personal time has become pretty tied up for the last ~6 months or more now. :-1:

Anyway, I'm going to consider this a duplicate of #463. The descriptions sound different but the underlying causes and fixes are the same: achieving proper `MouseEvent` propagation cross-browser.

**Comment by LeonidOvcharenko** [comment]
For those who will face with this problem, here is a workaround:
```
var els = $('tr td.copy');
var clip = new ZeroClipboard(els);
els.on('mouseleave', function(e){
   simulant.fire(e.target, 'mouseover', e.originalEvent);
});
```
where `simulant` is library https://github.com/Rich-Harris/simulant firing native `mouseover` event.

**Comment by Fental** [comment]
If user move the mouse too fast, the flash object has no enough time to move in place.
It just fired the mouse events which were attached to the DOM, so the flash was focus but not blur normally.

To fix this bug, I try to use  ```relatedTarget```.
Add codes in function ```_addMouseHandlers```

```javascript
var _elementMouseOut = function(event) {
    if (!(event || (event = _window.event))) {
      return;
    }
    _suppressMouseEvents(event);
    if (event.relatedTarget && $(event.relatedTarget).attr('id') !== _globalConfig.swfObjectId) {
      ZeroClipboard.blur();
    }
  };
  // Add the `mouseover` handler function
  element.addEventListener("mouseover", _elementMouseOver, false);

  // Add other mouse event handler functions
  element.addEventListener("mouseout", _elementMouseOut, false);
  element.addEventListener("mouseenter", _suppressMouseEvents, false);
  element.addEventListener("mouseleave", _elementMouseOut, false);
  element.addEventListener("mousemove", _suppressMouseEvents, false);

  // Save these function references to a global variable
  _mouseHandlers[element.zcClippingId] = {
    mouseover: _elementMouseOver,
    mouseout: _elementMouseOut,
    mouseenter: _suppressMouseEvents,
    mouseleave: _elementMouseOut,
    mousemove: _suppressMouseEvents
  };
```
